homeassistant/components/hive/strings.json

```diff
@@ -114,5 +114,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "import_error": {
+      "title": "YAML import failed for Hive",
+      "description": "Trying to import your YAML configuration is has failed\n\n Error {error}\n\nPlease remove your YAML configuration for Hive and continue [set up the integration](/config/integrations/dashboard)"
```

Comment:
I'm not sure if that is the right approach for this integration?

It seems like there are a couple of cases that can occur in this integration, for example, no internet connection available or incorrect code.

Is is really needed to remove the YAML configuration to fix that? Or should it be retries again when it becomes available?

I know you try to resolve this in a generic way, but that is the approach we tried to avoid using ADR-0021.

../Frenck

Reply:
I was more thinking that if an import fails it will leave the integration in the UI for the user to continue there.
Looking at the slightly more complex ones I wonder if we should not limit ourselves at least to create the issues than also implement retries etc. and then point the user to the right place to continue.

I wasn't trying to make this too generic. More to have a conceptual ok so we can then do it proper with the proper errors etc.